app/assets/stylesheets/wpcc/section/_details.scss

```diff
@@ -1,13 +1,15 @@
-.details {
-  @include column(12);
+.details__content {
```

Comment:
Same situation as above, will conditionally show/hide the contents of the step.  If I add this now without the JS logic all of the contents would be hidden.